tests/miscTests/ValidateLocalInfoXML.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * 
+ */
+function libxml_display_error($error)
```

Comment:
this function is the same as `libxml_display_error` in `htdocs/web_portal/GOCDB_monitor/validate_local_info_xml.php` can this function be replaced with a suitable `require` statement?